### DISCUSSION TITLE
Hourly ozone data from the atmosphere is not being read by CLM5

### DISCUSSION THREAD

**Discussion by jyoti1singh** [ESCOMP/CTSM#2287] [Q&A]
I have created two cases to run CLM5 to see the impact of ground ozone on crops and plants. I am using
ctsm5.1.dev113 in which Adriana Foster has added the capability using which CLM can read ozone data from the atmosphere. I have two cases, /glade/work/jyotis/O3_CLM5/O3_cases/b.e21.BW.f09_g17.SSP245-TSMLT-GAUSS-DEFAULT.006_hourlyO3
and /glade/work/jyotis/O3_CLM5/O3_cases/ctsm5.1.dev113_b.e21.BWSSP245cmip6.f09_g17.CMIP6-SSP2-4.5-WACCM.2060-2065.hourlyO3. To compute the ozone damage I added o3_veg_stress_method= 'stress_lombardozzi2015' in user_nl_clm and hourly ozone data was added in user_nl_datm_streams. Both cases ran successfully without any errors. However, later upon checking the value of history variables O3UPTAKESUN, O3UPTAKESHA, and ATM_O3 was found to be zero. Hence, CLM5 is not taking the atmospheric ozone as input.

I am not sure where might be the problem.

**Comment by ekluzek** [comment]
@adrifoster has been working with @jyoti1singh in regard to this and it seems to be resolved now.
